libs/partners/google-vertexai/langchain_google_vertexai/chat_models.py

```diff
@@ -274,13 +276,48 @@ def _parse_response_candidate(response_candidate: "Candidate") -> AIMessage:
         function_call = {"name": first_part.function_call.name}
 
         # dump to match other function calling llm for now
-        function_call["arguments"] = json.dumps(
-            {k: first_part.function_call.args[k] for k in first_part.function_call.args}
-        )
+        arguments = {}
+        for arg in first_part.function_call.args:
+            arg_value = first_part.function_call.args[arg]
+            if isinstance(arg_value, RepeatedComposite):
+                result = _convert_repeatedcomposite(arg_value)
```

Comment:
it looks a little bit excessive to me.
maybe smth like `first_part_dict = proto.Message.to_dict(first_part)` would be more readable / easier to grasp, wdyt?